daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -1233,10 +1236,11 @@ object Main extends Logging {
           case Some(fileName) => new FileInputStream(fileName)
         }
 
-        val rc: ExitCode.Value = processor match {
+        val rc = processor match {
           case Some(proc) if (proc.isError) => ExitCode.UnableToCreateProcessor
           case None => ExitCode.UnableToCreateProcessor
           case Some(processor) => {
+            Assert.invariant(processor.isError == false)
```

Comment:
Make sure to remove the `case Some(proc) if (proc.isError)` line, that line shouldn't be possible if you've made the above change correctly. Also, note that a similar line occurs in multiple places in the file. Make sure to remove all places where we proc.isError is true. The only place that should be possible is in the createProcessorFrom* functions.